app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -0,0 +1,13 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+export const calculateCreatedAgo = date => {
+  dayjs.extend(relativeTime);
```

Comment:
Create a singleton for this and move this to `lib/dayjs.js`. The `dayjs` instance is global. So once you extend it, whenever you import `dayjs` from the official module, it will have the extended properties. Similar to `axios`.